### PR TITLE
fix: tooltip doc sample

### PR DESCRIPTION
**Pull request by scomea**
# Description
Tooltip doc sample wasn't working so revised read me to point at component explorer examples.  Also addressed a similar situation with data grid.  

## Motivation & context
Samples should work.

## Issue type checklist
- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**
- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING

### CODE REVIEW

File: packages/web-components/fast-foundation/src/tooltip/README.md [hunk 1]
```diff
@@ -10,14 +10,24 @@ The `fast-tooltip` component is used provide extra information about another ele
 ## Usage
 
 ```html live
```
Comment (nicholasrice) [inline packages/web-components/fast-foundation/src/tooltip/README.md:12]: should `html` be `tsx`?

Comment (scomea) [reply packages/web-components/fast-foundation/src/tooltip/README.md]: Indeedy, corrected this same thing in data grid sample as well.

**Comment by EisenbergEffect** [comment]
I'm concerned about any form of React showing up in the docs. We shouldn't have that in the code samples. How is this going to appear?

File: sites/site-utilities/src/definitions/fast-components/fast-tooltip.definition.ts [hunk 2]
```diff
@@ -1,62 +0,0 @@
-import { WebComponentDefinition } from "@microsoft/fast-tooling/dist/esm/data-utilities/web-component";
-import { DataType } from "@microsoft/fast-tooling";
-
```
Comment (nicholasrice) [inline sites/site-utilities/src/definitions/fast-components/fast-tooltip.definition.ts:3]: This isn't needed?

Comment (scomea) [reply sites/site-utilities/src/definitions/fast-components/fast-tooltip.definition.ts:3]: No, no other component has one here.   I think it was a holdover from tooltip being added to the depot while the definition file location was changed.

**Review by chrisdholt — APPROVED** [review]

**Review by nicholasrice — APPROVED** [review]

**Pull request merged**